src/main/external-resources/renderers/Sony-PlayStationVita.conf

```diff
@@ -21,7 +21,7 @@ SeekByTime = exclusive
 DLNALocalizationRequired = true
 TranscodeVideo = MPEGTS-H264-AAC
 TranscodeAudio = WAV
-H264Level41Limited = true
+H264LevelLimit = 4.1
```

Comment:
@SubJunk Since this was introduced in 6.5.x (?) and the default set to 4.1, I assume that was without actually checking the capabilities of the renderers. If my assumption is correct, I see no reason to "preserve" this arbitrary value. As you can see from #1418 this has already created problems for users, and for every report I assume there are hundreds that doesn't report it. In my opinion, it would be better to define those limitations that has actually been verified and leave the rest undefined.